dluhc-component-library/src/components/maps/MapComponent.tsx

```diff
@@ -28,40 +28,54 @@ interface DrawingMapProps {
   circleFillColor?: string;
 }
 
+const DrawingMapProperties = {
```

Comment:
Can we rename this to `DEFAULT_DRAWING_PROPERTIES` since its a constant and not a drawing map.